issue_15/script.js

```diff
@@ -0,0 +1,87 @@
+const showBtn = document.getElementById('js-show-btn');
+const modal = document.getElementById('js-modal');
+const closeBtn = document.getElementById('js-close-btn');
+const submitBtn = document.getElementById('js-submit-btn');
+
+
+function startLoading() {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function removeLoading() {
+    document.getElementById('js-loading').remove();
+}
+
+function renderList(resolvedData) {
+    const ul = document.getElementById('js-ul');
+    const fragment = document.createDocumentFragment();
+    for (const item of resolvedData) {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+
+        a.href = item.a;
+        a.textContent = item.text;
+        img.src = item.img;
+        img.alt = item.alt;
+
+        fragment.appendChild(li).appendChild(a).appendChild(img);
+    }
+    ul.appendChild(fragment);
+}
+
+async function fetchData() {
+    try {
+        const URL = 'https://api.json-generator.com/templates/NluaaELSLhVe/data?access_token=zjrk7bmqgvptk4dcs8p8bhs5n9sk7m1ecs65gvq4';
+        const response = await fetch(URL);
+        if (!response.ok) {
+            console.error(`${response.status}:${response.statusText}`);
+        }
+        const result = await response.json();
+        return result.data;
+    } catch (error) {
+        console.error(error);
+    }
+}
+
+async function init(num, text) {
+    startLoading();
+    let listData;
+    try {
+        console.log(num, text);
+        listData = await fetchData();
+        if (!listData) {
+            return;
+        }
+        renderList(listData);
+    } catch (error) {
+        console.error(error);
+    } finally {
+        removeLoading();
+    }
+}
+
+showBtn.addEventListener('click', () => {
+    modal.classList.add('show');
+    showBtn.classList.add('hide');
+});
+
+closeBtn.addEventListener('click', () => {
+    modal.classList.remove('show');
+    showBtn.classList.remove('hide');
+});
+
+submitBtn.addEventListener('click', () => {
+    const inputNumber = document.getElementById('js-input-number').value;
+    const inputText = document.getElementById('js-input-text').value;
+    if (inputNumber == '' || inputText == '') {
```

Comment:
IMO
[Using Strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality) is a more secure code
```suggestion
    if (inputNumber === '' || inputText === '') {
```

Reply:
Thanks for pointing that out.
I had not paid much attention to this in the past, so this was a good opportunity to learn.
I decided to use the method described in [other reviews](https://github.com/yoshikawa-akr/js-lessons/pull/15#discussion_r986913431) this time, but I will try to be aware of it in the future :raised_hands: